From patchwork Fri Apr 17 12:27:12 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Fabio Estevam <festevam@gmail.com>
X-Patchwork-Id: 1272208
X-Patchwork-Delegate: sbabic@denx.de
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=2a01:238:438b:c500:173d:9f52:ddab:ee01; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=Obn1Ycw6; dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de
 [IPv6:2a01:238:438b:c500:173d:9f52:ddab:ee01])
 (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
 key-exchange X25519 server-signature RSA-PSS (4096 bits) server-digest
 SHA256) (No client certificate requested)
 by ozlabs.org (Postfix) with ESMTPS id 493b7d0GvNz9sR4
 for <incoming@patchwork.ozlabs.org>; Fri, 17 Apr 2020 22:31:56 +1000 (AEST)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
 by phobos.denx.de (Postfix) with ESMTP id B111B81C78;
 Fri, 17 Apr 2020 14:31:47 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de; dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="Obn1Ycw6";
 dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id 9D27A81C77; Fri, 17 Apr 2020 14:31:00 +0200 (CEST)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE autolearn=ham
 autolearn_force=no version=3.4.2
Received: from mail-qt1-x841.google.com (mail-qt1-x841.google.com
 [IPv6:2607:f8b0:4864:20::841])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id E839E81C77
 for <u-boot@lists.denx.de>; Fri, 17 Apr 2020 14:26:56 +0200 (CEST)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=festevam@gmail.com
Received: by mail-qt1-x841.google.com with SMTP id l13so1715491qtr.7
 for <u-boot@lists.denx.de>; Fri, 17 Apr 2020 05:26:56 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025; 
 h=from:to:cc:subject:date:message-id:in-reply-to:references;
 bh=Yz3tmmzY0y8w44Gsa9P6xR/Sz6lAJi28zFytmrf/4F8=;
 b=Obn1Ycw6rRh7tbOFfpJ2IIgAW3NS09vOYQ+Bno5ZqLElYgmHZ+aIoJMViDdrTOSf73
 MqznEmhYo46DMzZerFjImkD1mP16Rp3+MPpMJVBb9m8ae7zwxiuB0Djn6VP6b0LlmZNX
 EgDgG2MunWs0Af8EeOVUFCMgrSr31TVIAltV1lwQ250eqhi+CNEkwbu6LReASA0W8hqM
 W/K/ORYmysI/Uy3gcFZ66xJuO7mp7MKO4RyNh/9xHMxA9wC1oa9+w/kXDDOzX7yiTNZm
 AvvETqylxNWIQWcOOZ/IAeRjB0iqaY7XGimgCQ4XTdSEQinQdHOKxktF3ufRs1EUUdaf
 75Xw==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references;
 bh=Yz3tmmzY0y8w44Gsa9P6xR/Sz6lAJi28zFytmrf/4F8=;
 b=jkbEIwzMg7wtMAqumOLXPmSVsE8mpwcwmVVyazGqJPtX2P313S7sAP3cXzAzaAVkvf
 0P8BUhQ+R/21EfZMR3T4udH/oY93XWJhrgdJu0ClA2rO0Wux8l9EYOee06uWxvdBd5bG
 /sv26HGaq1LJlHHmsaWqSzjUPAqxoO1hNFRYKYtxH7dHpBO6jHlwjbZxTFJodL4mEIfc
 E136PeaJRD4qvFSxWya9Z68zlO7yoNb20LvrIApd/+/MrjHrfYkK9h3S0Dnemzl5qOhg
 WhJpuyxdeoDIJRQQPat+uyZeiB7H5KKonhEx0Sy7ZZQIG9qktlSRbNcEsVNoqwnzxQUc
 ZbXg==
X-Gm-Message-State: AGi0PuYK/1FP4wdgAiG9nzpPyCzJXcVP5l02Ym2GUcTfClQACUKxz0sa
 dwHHN0sUSOgHsLfuoj2Nbb1hNT+r
X-Google-Smtp-Source: APiQypLq78TR3pZF9eTcxq6m4el0vlaakGrPpfeGB9OnTjMdfTXMr8kYL3w70szL8m9W7VfFBe4ALA==
X-Received: by 2002:aed:2b64:: with SMTP id p91mr2558197qtd.202.1587126406763; 
 Fri, 17 Apr 2020 05:26:46 -0700 (PDT)
Received: from localhost.localdomain ([2804:14c:482:271:a8cc:37bd:62f6:da03])
 by smtp.gmail.com with ESMTPSA id
 m66sm16997985qkf.42.2020.04.17.05.26.44
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Fri, 17 Apr 2020 05:26:45 -0700 (PDT)
From: Fabio Estevam <festevam@gmail.com>
To: sbabic@denx.de
Cc: hs@denx.de, derek@ihtfp.com, u-boot@lists.denx.de,
 Fabio Estevam <festevam@gmail.com>
Subject: [PATCH v2 4/5] wandboard: Remove CONFIG_DISPLAY_BOARDINFO_LATE
Date: Fri, 17 Apr 2020 09:27:12 -0300
Message-Id: <20200417122713.30574-4-festevam@gmail.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20200417122713.30574-1-festevam@gmail.com>
References: <20200417122713.30574-1-festevam@gmail.com>
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.30rc1
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.2 at phobos.denx.de
X-Virus-Status: Clean

Since we are using revD1 device tree for all board revisions, the
following can be seen on a revB1 board:

Model: Wandboard i.MX6 Quad Board revD1
Board: Wandboard rev B1

To avoid such confusing messages, disable CONFIG_DISPLAY_BOARDINFO_LATE.

Signed-off-by: Fabio Estevam <festevam@gmail.com>
---
Changes since v1:
- None

 configs/wandboard_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/wandboard_defconfig b/configs/wandboard_defconfig
index ee70758f3a..f9a5fe479b 100644
--- a/configs/wandboard_defconfig
+++ b/configs/wandboard_defconfig
@@ -29,7 +29,6 @@ CONFIG_BOOTCOMMAND="run findfdt; run finduuid; run distro_bootcmd"
 CONFIG_SYS_CONSOLE_IS_IN_ENV=y
 CONFIG_SYS_CONSOLE_OVERWRITE_ROUTINE=y
 # CONFIG_DISPLAY_BOARDINFO is not set
-CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_BOUNCE_BUFFER=y
 CONFIG_BOARD_EARLY_INIT_F=y
 CONFIG_SPL_SEPARATE_BSS=y
